Base/Python/slicer/util.py

```diff
@@ -156,7 +156,7 @@ def showStatusMessage(message, duration = 0):
   mw.statusBar().showMessage(message, duration)
   return True
 
-def findChildren(widget=None, name="", text="", title="", className=""):
```

Comment:
It's the only reliable way that I found to get a widget from a CLI GUI. Yes, only useful for testing/prototyping.